### PR TITLE
Begin using requirements.txt

### PR DESCRIPTION
**Pull request by SSPJ**
What it says on the tin.

> The requirements.txt is used by Cloud.gov. It is needed to work around a bug in the CloudFoundry buildpack version of Pipenv that breaks on installing [packages] from a git repository.

### CODE REVIEW

**Review by loganmeetsworld — APPROVED** [review]
Looks great! Thanks for updating documentation.

File: src/requirements.txt
Comment (neilmb) [inline src/requirements.txt:1]: **question**: I don't see a git URL in here that necessitated adding the requirements.txt. Does that come in a separate PR?

**Review by neilmb — APPROVED** [review]
LGTM.

File: src/requirements.txt
Comment (SSPJ) [reply src/requirements.txt:1]: Yes. I'm currently working from the CISA fork in my sandbox and I don't want to deploy (possibly broken) changes to other environments.

**Pull request merged**